pkg/controller/migplan/validation.go

```diff
@@ -28,14 +29,20 @@ const (
 	InvalidDestinationCluster      = "InvalidDestinationCluster"
 	NsNotFoundOnSourceCluster      = "NamespaceNotFoundOnSourceCluster"
 	NsNotFoundOnDestinationCluster = "NamespaceNotFoundOnDestinationCluster"
-	StorageEnsured                 = "StorageEnsured"
+	// Warn
+	PvInvalidAction = "PvInvalidAction"
```

Comment:
I see that this is a warning. Do we select a default action in this case?

Reply:
No, it's a warning because it requires the user to resolve (choose an action).  It prevents the `Ready` condition but isn't something badly broken.  We _may_ be able to choose another category to make this clearer but nothing came to mind.

Reply:
Got it. To me, `Warning` in the context of software is something that can be optionally ignored if a user / dev chooses to do so. 

If this stops the Migration from being able to run it seems like more than a warning, at least by my definition.

Reply:
Fair enough.  Perhaps Category=`Incomplete` would be better.  Other ideas?

Reply:
Adding a `Critical` condition in another PR which will be used like `Error` is now for validation errors.  Then will change this to use `Error`.